packages/obonode/obojobo-chunks-materia/adapter.js

```diff
@@ -0,0 +1,39 @@
+import cloneProps from 'obojobo-document-engine/src/scripts/common/util/clone-props'
+import TextGroupAdapter from 'obojobo-document-engine/src/scripts/common/chunk/text-chunk/text-group-adapter'
+import TextGroup from 'obojobo-document-engine/src/scripts/common/text-group/text-group'
+import IFrameControlTypes from 'obojobo-chunks-iframe/iframe-control-types'
+
+const propsList = ['controls', 'height', 'icon', 'src', 'widgetEngine', 'width']
+
+export default {
+	construct(model, attrs) {
+		// process the caption text
+		if (attrs && attrs.content && attrs.content.textGroup) {
+			model.modelState.textGroup = TextGroup.fromDescriptor(attrs.content.textGroup, 1, {})
+		} else {
+			model.modelState.textGroup = TextGroup.create(1, {})
+		}
+
+		model.setStateProp('src', null)
+		model.setStateProp('width', null, p => parseInt(p, 10) || null)
+		model.setStateProp('height', null, p => parseInt(p, 10) || null)
+		model.setStateProp('widgetEngine', null)
+		model.setStateProp('icon', null)
+		// iframe controls
+		model.setStateProp('controls', [IFrameControlTypes.RELOAD])
```

Comment:
Since there's no way to set these options in the visual editor UI, I'm going to remove this and force reload as the only option. In the future we can add in controls in the GUI, and then we can change this to accept values